lib/populate/extract-csv.js

```diff
@@ -29,8 +36,41 @@ function extractDate(row) {
   }
 }
 
+function extractValidAltKeys(row) {
+  const altKeysArr = []
+  const altKeys = Object.keys(row).filter(r => validAltLang.has(r))
+  altKeys.forEach(k => altKeysArr.push(k))
+
+  return altKeysArr
+}
```

Comment:
Nous pourrions simplifier le code ici.

```suggestion
function extractValidAltKeys(row) {
  return Object.keys(row).filter(r => validAltLang.has(r))
}
```